tests/fsharp/update.base.line.with.actuals.fsx

```diff
@@ -0,0 +1,27 @@
+open System.IO
+
+// this script is usefull for tests using a .bsl file (baseline) containing expected compiler output
+// which is matched against .vserr or .err file aside once the test has run
+// the script replaces all the .bsl/.bslpp with either .err or .vserr
+
+let directories =
+    [
+      "typecheck/sigs"
+      "typeProviders/negTests"
+    ]
+    |> List.map (fun d -> Path.Combine(__SOURCE_DIRECTORY__, d) |> DirectoryInfo)
+
+let extensionPatterns = ["*.err"; "*.vserr"]
```

Comment:
You should also move `*.il` --> `*.il.bsl` under `fsharpqa\Source`, e.g.

![image](https://user-images.githubusercontent.com/7204669/41047231-fe157624-69a3-11e8-8919-8a439cb3837e.png)

Reply:
Make it a unified script under `tests\scripts`?